### PR TITLE
Update to the topic 1.1

### PR DESCRIPTION
**Pull request by Gowtham-GitHub**
1.Instance profile added
2.Minor changes to the S3 CLI commands

### CODE REVIEW

File: saa/1.1. Interacting with services - AWS CLI & SDK.md [hunk 1]
```diff
@@ -34,19 +34,20 @@
...
     - EC2 portal -> Right click on instance -> Settings -> Attach IAM role
 - **S3 CLI**
-  - `aws s3 ls`: Lists all buckets available
-  - `aws s3 ls s3://bucketname`: Lists contents of the bucket
-  - `aws s3 cp s3://bucketname/pic.jpg pic.jpg`: Copy a local file, or S3 objects to your computer or from your computer.
-  - `aws s3 mb s3://newbucket`: create a new bucket
-  - `aws s3 rb s3://newbucket`: delete existing bucket
+  - `aws s3 ls --region us-east-1`: Lists all buckets available
+  - `aws s3 ls s3://bucketname --region us-east-1`: Lists contents of the bucket
+  - `aws s3 cp s3://bucketname/pic.jpg --region us-east-1`: Copy a local file, or S3 objects to your computer or from your computer.
```
Comment (undergroundwires) [inline saa/1.1. Interacting with services - AWS CLI & SDK.md]: `cp` requires two file arguments maybe add back `pic.jpg` such as: `aws s3 cp pic.jpg s3://bucketname/pic.jpg`

File: saa/1.1. Interacting with services - AWS CLI & SDK.md [hunk 2]
```diff
@@ -34,19 +34,20 @@
       - If EC2 is shared -> other people may perform actions impersonating you
   - 💡 Better way: AWS IAM Roles
     - IAM Roles can be attached to EC2 instances.
-      - ❗ One EC2 instance can have one IAM role at a time
+      - ❗ One EC2 instance can have one IAM role at a time(but the same role can be attached to many other EC2 instances)
```
Comment (undergroundwires) [inline saa/1.1. Interacting with services - AWS CLI & SDK.md]: nice! can we have whitespace before the parenthesis or remove the parenthesis completely?

**Comment by undergroundwires** [comment]
Happy to see you again @Gowtham-GitHub 😀 Awesome contributions again. Commented on some small points.

File: saa/1.1. Interacting with services - AWS CLI & SDK.md (hunk 2 above)
Comment (Gowtham-GitHub) [reply saa/1.1. Interacting with services - AWS CLI & SDK.md]: Sure @undergroundwires we can have the blank space before the parenthesis.

File: saa/1.1. Interacting with services - AWS CLI & SDK.md (hunk 1 above)
Comment (Gowtham-GitHub) [reply saa/1.1. Interacting with services - AWS CLI & SDK.md]: When I made in a hurry, I missed it, glad you are here to note that.

**Comment by Gowtham-GitHub** [comment]
> 
> 
> Happy to see you again @Gowtham-GitHub 😀 Awesome contributions again. Commented on some small points.

More contributions are underway :-)

**Comment by Gowtham-GitHub** [comment]
> 
> 
> > Happy to see you again @Gowtham-GitHub grinning Awesome contributions again. Commented on some small points.
> 
> More contributions are underway :-)

**Pull request merged**